// Automatically Generated in 2017-06-12 22:33:47.453434
#ifndef MAVLINK_PHOENIX_MAVLINK2ROS_H
#define MAVLINK_PHOENIX_MAVLINK2ROS_H
#include <mavlink_phoenix/NOTIFICATION.h>
#include <mavlink_phoenix/HEARTBEAT.h>
#include <mavlink_phoenix/DEBUG.h>
#include <mavlink_phoenix/TELEMETRY.h>
#include <mavlink_phoenix/CONTROL_LIGHTS.h>
#include <mavlink_phoenix/CONTROL_COMMAND.h>
#include <mavlink_phoenix/IMU.h>
#include <mavlink_phoenix/ODOMETER_ABS.h>
#include <mavlink_phoenix/ODOMETER_RAW.h>
#include <mavlink_phoenix/ODOMETER_DELTA.h>
#include <mavlink_phoenix/ODOMETER_DELTA_RAW.h>
#include <mavlink_phoenix/ODOMETER.h>
#include <mavlink_phoenix/PROXIMITY.h>
#include <mavlink_phoenix/PARKING_LOT.h>
#include <mavlink_phoenix/CONFIG_REQUEST_COUNT.h>
#include <mavlink_phoenix/CONFIG_REQUEST.h>
#include <mavlink_phoenix/CONFIG_REQUEST_PARAMS.h>
#include <mavlink_phoenix/CONFIG_COUNT.h>
#include <mavlink_phoenix/CONFIG.h>
#include <mavlink_phoenix/CONFIG_PARAM_INT.h>
#include <mavlink_phoenix/CONFIG_PARAM_BOOL.h>
#include <mavlink_phoenix/CONFIG_PARAM_FLOAT.h>
#include <mavlink_phoenix/CONFIG_PARAM_SET_INT.h>
#include <mavlink_phoenix/CONFIG_PARAM_SET_BOOL.h>
#include <mavlink_phoenix/CONFIG_PARAM_SET_FLOAT.h>
#include <mavlink_phoenix/COMMAND.h>
#include <mavlink_phoenix/MAV_RAW_DATA.h>
#endif // MAVLINK_PHOENIX_MAVLINK2ROS_H
